tests/utils/exec/machine-pools.go

```diff
@@ -58,18 +59,34 @@ func (mp *MachinePoolService) Create(createArgs *MachinePoolArgs, extraArgs ...s
 	return nil
 }
 
-func (mp *MachinePoolService) Output() (mpName string, err error) {
+func (mp *MachinePoolService) Output() (MachinePoolArgs, error) {
```

Comment:
That's weird to return machinePool Args here. In future, args cannot meet the output requirement. Let's define an output struct.

Reply:
I added the MachinePoolOutput struct. @xueli181114 please, take a look is it what you asked for